node/blockchain/views/account_state.py

```diff
@@ -7,11 +8,24 @@
 from node.blockchain.serializers.account_state import AccountStateSerializer
 from node.blockchain.types import AccountNumber
 
+from .open_api_examples import account_state_open_api_example
+
 
 class AccountStateViewSet(RetrieveModelMixin, GenericViewSet):
     serializer_class = AccountStateSerializer
     queryset = AccountStateSerializer.Meta.model.objects.all()
 
+    @extend_schema(
+        summary='Retrieve Account state by ID',
+        examples=[account_state_open_api_example],
+    )
+    def retrieve(self, request, *args, **kwargs):
+        """
+        Account state, which defines a set of attributes about the current account state, such as account balance
```

Comment:
Can we add this description to the model (or serializer) instead, so it gets to the docs automatically? Or a least we reuse it from there?